PWGGA/GammaConv/macros/AddTask_GammaConvNeutralMesonPiPlPiMiNeutralMeson_MixedMode_pp.C

```diff
@@ -183,6 +184,7 @@ AliVEventHandler *inputHandler=mgr->GetInputEventHandler();
 
   AliAnalysisTaskNeutralMesonToPiPlPiMiNeutralMeson *task=NULL;
   task= new AliAnalysisTaskNeutralMesonToPiPlPiMiNeutralMeson(Form("GammaConvNeutralMesonPiPlPiMiNeutralMeson_%i_%i",neutralPionMode, trainConfig));
```

Comment:
running multiple pion selectors in same trains does not work in this case, since the output files for two different pion preselections will have the same name, causing an error